sequencing/vcf_query/Readme.md

```diff
@@ -1,21 +1,15 @@
 <!-- dx-header -->
 # vcf_query (DNAnexus Platform App)
 
-Calls "bcftools query" to extract annotations from the VCF file
+## What does this app do?
+Calls "bcftools query" to extract annotations from the VCF files (`*.vcf.gz`).
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+## What data are required for this app to run?
 
+This app requires the VCF file(s) (`*.vcf.gz`) along with its index file (`*.vcf.gz.tbi`) as an input. Also, as an required input, it takes in a string input consisting of the query that would be performed on the VCF file. This can contain fields of the VCF file in a tab delimited format. For example, the default query the app will perform is "%CHROM\\t%POS\\t%ID\\t%REF\\t%ALT\\t%QUAL\\t%INFO/QD\\t%INFO/AN\\t%INFO/AC[\\t%GT]\\n"
```

Comment:
"This app requires VCF file(s) (`*.vcf.gz`) along with its respective index file(s) (`*.vcf.gz.tbi`) as input. Also, as a required input, the app takes in a string input consisting of the query to be performed on the VCF file. This can contain fields of the VCF file in a tab-delimited format. For example, the default query the app will perform is `%CHROM\\t%POS\\t%ID\\t%REF\\t%ALT\\t%QUAL\\t%INFO/QD\\t%INFO/AN\\t%INFO/AC[\\t%GT]\\n`."